twistedchecker/functionaltests/maxlinelength.py

```diff
@@ -1,6 +1,6 @@
 # enable: C0301
 # max line length of Twsited projects should be 79
-print "this line is long long  long  long long long long long ends at column 80"
+print("this line is long long  long  long long long long long ends at column 80")
```

Comment:
It doesn't though - it ends at column 81. You removed one space and added two parentheses.  The fact that this doesn't cause anything to fail is cause for alarm :)